drivers/include/dfplayer.h

```diff
@@ -0,0 +1,751 @@
+/*
+ * Copyright 2019 Marian Buschsieweke
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_dfplayer DFPlayer Mini MP3 Player
+ * @ingroup     drivers_multimedia
+ * @brief       Driver for the DFPlayer Mini MP3 Player
+ *
+ * The DFPlayer Mini is a super cheap MP3 Player that can be controlled via
+ * UART. It can play files from SD card, USB storage devices and NOR flash.
+ * It has an integrated 3W amplifier that can be used to build a standalone
+ * mono speaker. Alternatively a stereo output can be used that can be
+ * directly connected to headphones, but needs a dedicated amplifier for
+ * connecting to passive speakers.
+ *
```

Comment:
Would be neat to have a pinout here
![DFPlayer Pinout](https://cdn.shopify.com/s/files/1/1509/1638/files/MP3_player_module_pinout_diagram.png)